docs/adrs/0003-bulk-import-ui.md

```diff
@@ -55,8 +55,17 @@ The reason for the App cell being an extension to code cells is the last point a
 
 ## Decision Outcome
 
+The Bulk import UI will be developed in a new cell type, based on the existing app cell.
+
+Other points discussed:
+* We considered using a different interface than a cell, but decided that changing that entire paradigm right now would be too heavy of a burden on the design and product teams and would push back development efforts further. There was also no clear consensus on whether that's a desired result.
```

Comment:
> but decided that changing that entire paradigm right now would be too heavy of a burden on the design and product teams and would push back development efforts further

I'm still not sure I believe this, but I'm in the minority apparently so *shrug*.